packages/EasyErrorHandler/src/Bridge/Laravel/Handler/Handler.php

```diff
@@ -0,0 +1,332 @@
+     * {@inheritdoc}
+     */
+    public function renderForConsole($output, Exception $exception): void
+    {
+        (new ConsoleApplication())->renderThrowable($exception, $output);
+
+        $this->renderTranslationToConsoleIfNeeded($output, $exception);
+
+        $this->renderValidationFailuresToConsoleIfNeeded($output, $exception);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function report(Exception $exception): void
+    {
+        if ($this->shouldReport($exception) === false) {
+            return;
+        }
+
+        $logLevel = $exception instanceof LogLevelAwareExceptionInterface ?
+            $exception->getLogLevel() :
+            LoggerInterface::LEVEL_ERROR;
+
+        $this->logger->exception($exception, $logLevel);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function shouldReport(Exception $exception): bool
+    {
+        foreach ($this->dontReport as $type) {
+            if ($exception instanceof $type) {
+                return false;
+            }
+        }
+
+        return true;
+    }
+
+    /**
+     * Returns a determined user message.
+     */
+    protected function determineUserMessage(Exception $exception): string
+    {
+        $userMessage = null;
+        $userMessageParams = [];
+        if ($exception instanceof TranslatableExceptionInterface) {
+            $userMessage = $exception->getUserMessage();
+            $userMessageParams = $exception->getUserMessageParams();
+        }
+
+        return $this->translateMessage($userMessage ?? $this->defaultUserMessage, $userMessageParams);
+    }
+
+    /**
+     * Returns the current datetime string.
+     */
+    protected function getCurrentDateTimeString(): string
+    {
+        return Carbon::now()->toIso8601ZuluString();
+    }
+
+    /**
+     * Returns an exception converted into an array.
+     *
+     * @return mixed[]
+     */
+    protected function getResponseData(Exception $exception): array
+    {
+        return $this->useExtendedResponse() ?
+            $this->getExtendedResponseData($exception) :
+            $this->getShortResponseData($exception);
+    }
+
+    /**
+     * Returns a response status code based on an exception.
+     */
+    protected function getResponseStatusCode(Exception $exception): int
+    {
+        if ($exception instanceof StatusCodeAwareExceptionInterface === false) {
+            return 500;
+        }
+
+        return $exception->getStatusCode();
+    }
+
+    /**
+     * Checks whether the extended response should be used.
+     */
+    protected function useExtendedResponse(): bool
+    {
+        return $this->config->get('app.debug', false) === true;
+    }
+
+    /**
+     * Returns a determined exception message.
+     */
+    private function determineExceptionMessage(Exception $exception): string
+    {
+        $message = $exception->getMessage();
+        $params = $exception instanceof TranslatableExceptionInterface ? $exception->getMessageParams() : [];
```

Comment:
I think that here we should check if the exception is translatable first, if not then return its message directly, this way we avoid calling the translator when not needed. The translator will potentially look in the filesystem which can be expensive.

Reply:
done